app/models/item.rb

```diff
@@ -33,7 +33,7 @@
 class Item < ApplicationRecord
   belongs_to :campaign, optional: true
   belongs_to :payment_intent, optional: true
-  belongs_to :purchaser, class_name: 'Contact'
+  belongs_to :purchaser, class_name: 'Contact', optional: true
 
   # Must have seller or project, but not both
```

Comment:
so one thing is that the items that are created by the allocation endpoint have a seller and project. we use the project as a holder for the campaign money, but the item needs a specific seller associated as well. does anyone happen to remember why we have an xor here? would it break anything if i removed this validation?

Reply:
It won't break anything! We had it because at the time of creation it only knows one or the other

Reply:
ok got it, thanks!

Reply:
Yeah, feel free to change this if things have changed! This was an initial assumption and we put these guards in as a safety net!